Gemfile.lock

```diff
@@ -176,7 +176,7 @@ GEM
     zeitwerk (2.4.2)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
Do you have an idea what impact this change might have?

Reply:
Not a clue, unfortunately. @hopsoft?